src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -45,18 +48,29 @@ let nbRow: number = 0;
     '[class.datagrid-row]': 'true',
     '[class.datagrid-selected]': 'selected',
     '[attr.aria-owns]': 'id',
+    '[style.overflow-y]': '"hidden"',
     role: 'rowgroup',
   },
   providers: [
     DatagridIfExpandService,
     { provide: IfExpandService, useExisting: DatagridIfExpandService },
     { provide: LoadingListener, useExisting: DatagridIfExpandService },
   ],
+  animations: [
+    trigger('expandAnim', [
+      transition('void => *', []),
```

Comment:
what is this empty transition for?

Reply:
To skip animation if initialized as expanded.